cli/__snapshots__/cli_spec.js

```diff
@@ -19,14 +19,15 @@ exports['shows help for open --foo 1'] = `
 
   Options:
 
-    -p, --port <port>             runs Cypress on a specific port. overrides any value in cypress.json.
-    -e, --env <env>               sets environment variables. separate multiple values with a comma. overrides any value in cypress.json or cypress.env.json
-    -c, --config <config>         sets configuration values. separate multiple values with a comma. overrides any value in cypress.json.
-    -d, --detached [bool]         runs Cypress application in detached mode
-    -P, --project <project path>  path to the project
-    --global                      force Cypress into global mode as if its globally installed
-    --dev                         runs cypress in development and bypasses binary check
-    -h, --help                    output usage information
+    -p, --port <port>                runs Cypress on a specific port. overrides any value in cypress.json.
+    -e, --env <env>                  sets environment variables. separate multiple values with a comma. overrides any value in cypress.json or cypress.env.json
+    -c, --config <config>            sets configuration values. separate multiple values with a comma. overrides any value in cypress.json.
+    -C, --config-file <config-file>  path to a config file to override the default cypress.json. set to "false" to skip specifying a config file.
```

Comment:
Is this required to be a JSON file? If so, this should be specified in description. Also would opt for use of word 'configuration' over 'config' in description as in previous description. Maybe:

"path to JSON file where configuration values are set. default path is './cypress.json'. pass 'false' to disable."